# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: dom/media/GraphDriver.cpp
# Commit: 02f63cb7c759
# Full Hash: 02f63cb7c759d53733e30b12d24d0c94f6e7473a
# Author: Karl Tomlinson <karlt+@karlt.net>
# Date: 2023-04-04 03:45:24
# Description:
#   Bug 1825913 remove invalid AudioStreamState::None assertion after cubeb_stream_start() failure r=pehrsons
#   
#   aaudio_stream_start() notifies CUBEB_STATE_ERROR asynchronously from another
#   thread.
#   
# ==============================================================================

diff -r 6cca93773e91 -r 02f63cb7c759 dom/media/GraphDriver.cpp
--- a/dom/media/GraphDriver.cpp	Mon Apr 03 18:59:49 2023 +0000
+++ b/dom/media/GraphDriver.cpp	Mon Apr 03 19:32:57 2023 +0000
@@ -762,7 +762,6 @@
   mAudioStreamState = AudioStreamState::Starting;
   if (cubeb_stream_start(mAudioStream) != CUBEB_OK) {
     NS_WARNING("Could not start cubeb stream for MTG.");
-    MOZ_ASSERT(mAudioStreamState == AudioStreamState::None);
     return false;
   }
 
